Adds a comment to hexagon()

sul3n3t opened the pull request.

johnemajor commented:
My original code had the following comment, FWIW... I don't know where it was lost.

def hexagon_from_btm_cell_by_coords(self, row: int, col: int) -> Tuple['TriangleCell','TriangleCell','TriangleCell','TriangleCell','TriangleCell','TriangleCell']:
        "Given the coordinates of an up facing cell, return the surrounding cells which will make "
        "A hexagon with the specified cell as the base"
        "the tuple will begin with the btm upward facing triangle at the base of the hex"
        "the next cell in the tuple will be the next triangle in the hexagon moving counterclockwise"
        "None will be entered for cells off the grid"

sul3n3t commented:
Thanks! I incorporated your comments and fixed the order. I didn't use it exactly as-is because it seems like it should be able to start with a down-facing cell.

sul3n3t commented:
I'm going to merge because it's just a comment PR. Feel free to make additional changes if you like.
